sass/_components.scss

```diff
@@ -654,4 +654,532 @@
         padding: $baseline;
         background: #363e4e;
     }
+}
+
+// Product Hero - 3x1
+//
+// .default - Default Hero
```

Comment:
I did some digging and it turns out you can render a default component without a class by doing this:

```sass
// Product Hero - 3x1
//
// default - Default
// .hide-cta - Hero without CTA Button
//
...
```

Means you dont end up with markup in the styleguide that has a default class on it